bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/MenuItem.java

```diff
@@ -1161,7 +1178,8 @@ LRESULT wmMeasureChild (long wParam, long lParam) {
 			 * NOTE: autoScaleUpUsingNativeDPI() is used to avoid problems
 			 * with applications that disable automatic scaling.
 			 */
-			struct.itemWidth = DPIUtil.scaleUp(5, nativeZoom);
```

Comment:
This has set the left margin of an entry before. now the margin appears to be zero and has to be added at some point. The first element is drawn exactly at x coordinate 0, which looks a bit odd:
![Screenshot 2024-08-19 130842](https://github.com/user-attachments/assets/9746d42a-e31e-43a6-9d08-22d076332ec7)

Here is a comparison with the existing spacing (on top). Not only the initial left margin is different, but also the other spacing seems to be different (larger than before):
![Screenshot 2024-08-19 130854](https://github.com/user-attachments/assets/be7a516c-2640-4500-bb98-9cb291744e2b)